extension/agenthealth/handler/stats/provider/flag.go

```diff
@@ -17,20 +17,15 @@ const (
 	flagGetInterval = 5 * time.Minute
 )
 
-type BoolFlag int
+type Flag int
 
 const (
-	FlagIMDSFallbackSucceed BoolFlag = iota
+	FlagIMDSFallbackSucceed Flag = iota
 	FlagSharedConfigFallback
 	FlagAppSignal
 	FlagEnhancedContainerInsights
 	FlagRunningInContainer
```

Comment:
So basically this is running in a container without container insights?

Reply:
If it's running in a container at all. It checks the `RUN_IN_CONTAINER` environment variable, which we set in our agent images (https://github.com/aws/amazon-cloudwatch-agent/blob/main/amazon-cloudwatch-container-insights/cloudwatch-agent-dockerfile/Dockerfile).

Reply:
So for tw customers, both FlagEnhancedContainerInishgts and FlagRunningInContainer should be true?

Reply:
Yes.